### PR TITLE
[REF] for compatitbility with pos_cache

### PR DESCRIPTION
**Pull request by KolushovAlexandr**

### CODE REVIEW

File: pos_product_available/static/src/js/pos.js [hunk 1]
```diff
@@ -3,19 +3,28 @@ odoo.define('pos_product_available.PosModel', function(require){
...
         initialize: function (session, attributes) {
-            var partner_model = _.find(this.models, function(model){ return model.model === 'product.product'; });
-            partner_model.fields.push('qty_available');
-            
-            return _super_posmodel.initialize.call(this, session, attributes);
+            var self = this;
+            models.load_fields('product.product',['qty_available']);
+//  pos_cache caches 'product.product' model and removes it from the models, what leads to incorrect product quantity displaying.
+//  for compatibility added a new 'product.product' model with the only field 'qty_available' to prevent incorrect displaying
+            models.load_models({
```
Comment (yelizariev) [inline pos_product_available/static/src/js/pos.js]: Try to load qty_available separately only when pos_cache in affect, i.e. when 'product.product' not in models list

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by Railia — APPROVED** [review]

**Review by Railia — CHANGES_REQUESTED** [review]
Change version in corresponding docs, files

**Review by Railia — APPROVED** [review]

**Review by yelizariev — APPROVED** [review]

File: pos_product_available/tests/test_quantities.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+import odoo.tests
+
+
+@odoo.tests.common.at_install(False)
```
Comment (yelizariev) [inline pos_product_available/tests/test_quantities.py]: Switch to True or add a comment why it cannot be True. Consult to @GabbasovDinar if needed

**Review by yelizariev — CHANGES_REQUESTED** [review]

**Review by yelizariev — APPROVED** [review]
code seems ok

**Pull request merged**